scout/adapter/mongo/variant_loader.py

```diff
@@ -488,7 +488,17 @@ def _load_variants(
                     bulk = {}
 
                 current_region = new_region
-                bulk[var_id] = variant_obj
+                if var_id in bulk:
```

Comment:
Perhaps a stupid question but I feel I have a limited understanding the bulks: what if the variant is in another bulk instead?

Reply:
Then the warning that was already in the code should work (database throws an exception, and each variant in the bulk is upserted separately to get to the issue and ensure that as many as possible have been inserted.